### DISCUSSION TITLE
中央真的沒有腐男腐女嗎 ; )

### DISCUSSION THREAD

**Discussion by woohsin** [1101-GS4719-C/Python-Community-GS4719-C#1948] [General]
高中因為同學推薦打開了第一部腐劇，之後就了解到什麼叫做一入腐門深似海，直接垂直入坑，但後來就不太看劇了，主要是我覺得真人的演員演技偏尬(尤其是台灣的演員:)但還是可以分養我心中各國top。
(都是很不專業的評論)
日本：美麗的他
描寫一對高中男學生的初戀故事，全劇貫穿著酸甜青澀的校園愛情，以及戀人既幽微又敏感的複雜心境。由萩原利久飾演的「平良一成」是一位患有口吃的高中生，平時相當沈默、在學校如同邊緣人，不過就在他上高三那年遇到了轉學來的清居奏（八木勇征飾），對這位帥氣的新學生一見鍾情，一段糾結的感情故事就此展開。
(原文網址: https://www.letsgojp.com/archives/517888/)
鏡頭都好美，國王真的長得很帥，而且人設好喜歡，兩個怪人哈哈
泰國：bad buddy
兩個一直敵對的家庭，雖然是鄰居卻不相往來(是不是很像羅密歐與茱麗葉?)。而兩個家裡的小孩Pat 和Pran，也在家長要求下，從小就是敵對競爭關係。在這樣的競爭中，兩人卻暗自彼此欣賞。高中時兩人合組樂團表演，被Pran母親看到，立刻安排Pran轉學。沒想到兩人又考上同一所大學，一個在工程系一個在建築系，兩個系又是學校裡的死敵。在這樣的長期敵對中，兩人卻發現除了彼此欣賞外，還成了彼此心裡重要的人。兩人無力抵抗外在環境的壓力，雖然逐漸走在一起，卻必須遮遮掩掩不被發現，當兩人情誼堅定之後還是東窗事發，來自兩方家長的壓迫、同儕間的敵對關係，都讓兩人深覺情感難以長久，決定分開…
(https://tonyliu0936.pixnet.net/blog/post/221176221-%E3%80%90%E7%9C%8B%E6%B3%B0%E5%8A%87%E3%80%91-%E6%95%B5%E4%BA%BA%E6%88%90%E6%83%85%E4%BA%BA%E3%80%8Abad-budd)
主角靠臉打天下，不愧是GMM太子爺，而且這部是少數我很喜歡女演員的一部
韓國：語意錯誤
劇情講述學校風雲人物「張宰英」（朴栖含 飾），和成績很好卻有點書呆子的邊緣人「秋尚宇」（朴宰燦 飾），陰錯陽差開始一起製作手遊。
兩人原本互看不順眼，尤其「張宰英」對於「秋尚宇」來說，就像是電腦程式中的BUG，偏偏在清除BUG過程中，兩人也因此展開一段甜蜜卻苦惱的校園愛情故事。
(https://www.cosmopolitan.com/tw/entertainment/movies/g39437113/semanticerror/)
雖然充滿著對資工系學生的刻板印象(我開學到現在真的沒看過資工學生穿格子襯衫...)但是畫面都拍得很精緻，而且有很多小彩蛋都很用心。然後學長退伍了，又可以看見他更多作品了！
後來部看劇是因印發現小說跟漫畫才是最對我胃口的，小說部份我尤其喜歡看修仙，看我的推薦就知道了。
天官賜福：天官賜福主要講述一個撿破爛專業的神仙謝憐，歷經三次飛升兩次被貶，在破任務收功德的過程中，與鬼王花城和其他小夥伴攜手打敗敵人的故事。
墨香三部曲我最喜歡的一本，花城真的很暖，害我花錢直接把小說都包回家了。而且裡面除了主角，許多配角的故事也都很精彩。
二哈和他的白貓師尊：
講述了滿手血腥的踏仙君重生回到了十餘年前的年少時代，重歷人生，發現了許多前世不曾覺察的秘密，明白了前世曾經錯失的感情。最終在其師尊的幫助下，阻止了塵世顛覆，拔除心魔，立身為善的故事。
基本上二刷開始我都只有看前1/2，後面是有一過分狗血了，但是前面看師尊真的會一直哭，雖然後面甜的也很好看，但我私心更愛虐的那些片段。

還有漫畫但我打字累了，等我有空再更新。

高中的時候班上一堆腐男腐女害我以為現在都是這樣的，直到上了大學被說很奇怪，還被用嫌棄的眼神看，才發現以前一直在自己的同溫層裡...
但大多數男生好像都很排斥，我大中央男校，害我在這裡不敢聊得太招搖，所以為什麼會那麼討厭阿...

**Comment by LinChloe** [comment]
不，其實腐女蠻多的。
之前在寢室問了才知道，大家都有在看，只是有沒有到狂熱的地步而已。
我有朋友只看BL小說，也有朋友拿文化幣去買18禁BL漫畫。
只是大家都不會明著說而已，大家應該都不會大聲宣揚自己的興趣吧。
男生的話確實喜歡的人很少。
話說你的文章很詳細，感覺的到真的是真愛，支持妳喔~
(我也喜歡墨香三部曲，魔道祖師讚~)

**Reply by woohsin** [reply]
對壓，大家都不會說出來，其實我也有漸漸找到同道中人啦~幸運:)
墨家的話花城是我這輩子看過最棒的男人(男鬼?

**Comment by 110101501** [comment]
自己身邊的朋友多少還是會有腐男腐女，但基本上也都不會明說。沒有談到相關的話題就不會主動透露。很多時候是以為對方沒有看，但實際上對方有看的狀況。
此外，被排斥的狀況目前倒沒有碰過，可能我周遭的朋友對此的接受度蠻高的！
